django_mailbox/models.py

```diff
@@ -386,10 +385,14 @@ def _process_message(self, message):
 
     def _process_save_original_message(self, message, msg):
         settings = utils.get_settings()
+        if six.PY3:
+            content = message.as_string().encode('ascii', 'surrogateescape')
```

Comment:
I'm feeling a little conflicted about this -- we'd be fundamentally changing the method of encoding stored files if we do this, no?  I might be misunderstanding what we're doing, here, though.